pkg/tests/ossm/initContainer_test.go

```diff
@@ -15,42 +15,73 @@
 package ossm
 
 import (
-	_ "embed"
-	"strings"
+	"fmt"
 	"testing"
 
-	"github.com/maistra/maistra-test-tool/pkg/util"
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/hack"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/pod"
 	"github.com/maistra/maistra-test-tool/pkg/util/test"
 )
 
 const (
-	initContainerNS         = "bookinfo"
-	initContainerGoldString = "init worked"
+	initContainerNS          = "bookinfo"
+	initContainerGoldString  = "[init worked]"
+	initContainerPodSelector = "app=sleep-init"
 )
 
-var (
-	//go:embed yaml/deployment-sleep-init.yaml
-	testInitContainerYAML string
-)
+func TestInitContainer(t *testing.T) {
+	test.NewTest(t).Id("T33").Groups(test.Full).Run(func(t test.TestHelper) {
 
```

Comment:
Maybe a short log message explaining the purpose of the log (or what it's actually verifying). According to the original commit, it's checking that sidecar injection does not remove the init container.